src/functionalTest/java/steps/ClaimStatusSteps.java

```diff
@@ -0,0 +1,85 @@
+package steps;
+
+import net.thucydides.core.annotations.Step;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+
+public class ClaimStatusSteps {
+
+    private String xml;
+    private Response response;
+    private int randomNumber;
+
+    @Step("Given the soap request body")
+    public String soapRequestBody() {
+        String xmlContent = "";
+        try {
+            xmlContent = new String(Files.readAllBytes(Paths.get(getClass().getClassLoader().getResource("statusUpdate.xml").toURI())));
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
```

Comment:
This code for reading a file is duplicated in the IssueWarrant and MCOLJudgment classes.  It would be better to put it in a method in a separate utility class and call that instead.

In the event of an exception being raised it would be better to fail the test rather than handling it and carrying on as is done here.  This would make it easier to debug.  In JUnit you can explicitly fail a test using `org.junit.jupiter.api.Assertions.fail` - I don't know if Serenity offers an equivalent.  If not you could just let the method throw the exception by adding `throws <exception class name>` to the signature.